pypfb/__main__.py

```diff
@@ -139,20 +132,30 @@ def main():
         avro_schema.write(args.output)
 
     elif args.cmd == "json2pfb":
-        PFBFile.from_json(
+        Gen3PFB.from_json(
             args.schema, args.dir, args.output, args.program, args.project
         )
 
+    elif args.cmd == "show":
+        if args.schema:
+            print(Gen3PFB(args.input).read_metadata())
+        else:
+            limit = args.limit if args.limit != -1 else None
+            for r in itertools.islice(Gen3PFB(args.input).read_records(args.input), limit):
+                print(r)
     elif args.cmd == "make":
-        make_record(args.input, args.node)
+        Gen3PFB(args.input).make_record(args.node)
 
     elif args.cmd == "add":
-        add_record(args.PFB_file, args.JSON_file)
+        Gen3PFB(args.PFB_file).add_record(args.JSON_file)
 
     elif args.cmd == "rename":
         if args.rename == "node":
-            rename_node(args.input, args.output, args.name_from, args.name_to)
+            Gen3PFB(args.input).rename_node(args.output, args.name_from, args.name_to)
+        elif args.rename == "enum":
+            Gen3PFB(args.input).rename_field_enum(args.output, args.field_name, args.val_from, args.val_to)
+
 
 
 if __name__ == "__main__":
-    main()
+    main()
```

Comment:
Add newline in the end of file.

Reply:
This can be changed later with autoformatting. 👍

Reply:
fixed in 395749242c00d4562e524dd8c22e3c2ea8887231